src/components/expandableSection/index.tsx

```diff
@@ -25,6 +26,12 @@ export type ExpandableSectionProps = {
    * action for when pressing the header of the expandableSection
    */
   onPress?: () => void;
+  /**
+   * Set a minimum height for the expandableSection
+   * If the children height is less than the minHeight, the expandableSection will collapse to that height
+   * If the children height is greater than the minHeight, the expandableSection will result with only the children rendered
```

Comment:
No clear "will result with only the children rendered". Do you mean "no height limit will apply"?

Reply:
Added `(sectionHeader will not be rendered)`